libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -1068,7 +1068,15 @@ bool ExpressionCompiler::visit(MemberAccess const& _memberAccess)
 			solAssert(false, "Invalid member access to integer");
 		break;
 	case Type::Category::Function:
-		solAssert(!!_memberAccess.expression().annotation().type->memberType(member),
+		if (member == "selector")
+		{
+			FunctionType const& type = dynamic_cast<FunctionType const&>(*_memberAccess.expression().annotation().type);
+			utils().popStackElement(type);
```

Comment:
Why not `m_context << Instruction::SWAP1 << Instruction::POP;`? (and then perhaps a shift left, not sure about that)

Reply:
So the stack should have `<address> <signature>`.